frontend/public/components/routes.tsx

```diff
@@ -186,13 +188,47 @@ const calcTrafficPercentage = (weight: number, route: any) => {
   return `${percentage.toFixed(1)}%`;
 };
 
+const getIngressStatusForHost = (hostname, ingresses) => {
+  return _.find(ingresses, {host: hostname});
+};
+
+const showCustomRouteHelp = (ingress, annotations) => {
+  if (!ingress || !_.some(ingress.conditions, {type: 'Admitted', status: 'True'})) {
+    return false;
+  }
+
+  if (_.get(annotations, 'openshift.io/host.generated') === 'true') {
+    return false;
+  }
+
+  if (!ingress.host || !ingress.routerCanonicalHostname) {
+    return false;
+  }
+
+  return true;
+};
+
 const RouteTargetRow = ({route, target}) => <tr>
   <td><ResourceLink kind={target.kind} name={target.name} namespace={route.metadata.namespace} title={target.name} /></td>
   <td>{target.weight}</td>
   <td>{calcTrafficPercentage(target.weight, route)}</td>
 </tr>;
 
-const RouteIngressStatus: React.SFC<RouteIngressStatusProps> = ({ingresses}) =>
+const CustomRouteHelp = ({host, canonicalHostname, id, learnMoreLink=null}) => <Popover id={id}
+  headerContent={<span>Custom route</span>}
+  bodyContent={
+    <div>
+      <p>To use a custom route, you must update your DNS provider by creating a canonical name (CNAME) record.
+      Your CNAME record should point to your custom domain
+      <span className="co-m-route-custom-hostname"> {host}</span>, to the OpenShift canonical router hostname,
```

Comment:
Why not `<strong>` and avoid this custom class? It's what we use elsewhere

Reply:
fixed